routers/repo/issue.go

```diff
@@ -515,7 +519,36 @@ func getFileContentFromDefaultBranch(ctx *context.Context, filename string) (str
 	return string(bytes), true
 }
 
-func setTemplateIfExists(ctx *context.Context, ctxDataKey string, possibleFiles []string) {
+func setTemplateIfExists(ctx *context.Context, ctxDataKey string, possibleDirs []string, possibleFiles []string) {
+	if ctx.Query("template") != "" {
+		for _, dirName := range possibleDirs {
+			content, found := getFileContentFromDefaultBranch(ctx, path.Join(dirName, ctx.Query("template")))
```

Comment:
I'd call this template content here - calling it content seems bind to cause confusion.